backends/verilog/verilog_backend.cc

```diff
@@ -1044,16 +1044,21 @@ void dump_cell_expr_print(std::ostream &f, std::string indent, const RTLIL::Cell
 void dump_cell_expr_check(std::ostream &f, std::string indent, const RTLIL::Cell *cell)
 {
 	std::string flavor = cell->getParam(ID(FLAVOR)).decode_string();
+	std::string indent_name = indent;
+	if (cell->name.isPublic()) {
+		indent_name += stringf("%s: ", id(cell->name).c_str());
+	}
```

Comment:
I would leave `indent` alone for consistency with the rest of the file, since you're changing all of the lines below anyway.